src/main/java/TurnUseCases/BuildUseCase/BuildBuildingInputBoundary.java

```diff
@@ -0,0 +1,33 @@
+package TurnUseCases.BuildUseCase;
+
+import GameEntities.Player;
+import GameEntities.Tiles.ColorPropertyTile;
+import GameEntities.Tiles.Property;
+
+public interface BuildBuildingInputBoundary {
```

Comment:
Do we want BuildBuilding use case to also have methods for selling houses and hotels?